### DISCUSSION TITLE
How to search for a particular commit

### DISCUSSION THREAD

**Discussion by julienroyd** [community/community#24773] [Apps, API and Webhooks]
<p>Hi,</p>
<p>I want to find a particular commit in a repository’s page to see if there are associated pull requests.</p>
<p>How do I do that? I have the commit hash but I can’t find any search-bar to navigate to that commit’s page?</p>
<p>Thanks!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>I’m not sure about the search, but if you have the commit ID you can construct the URL for the commit page like this:</p>
<p><code>https://github.com/{user}/{repo}/commit/{commit-id}</code></p>

**Reply by david-sharp-REED** [reply]
Nice lateral thinking. Thanks for that

**Comment by julienroyd** [comment]
<p>Thanks that works as a workaround</p>

**Comment by iliarunnrai** [comment]
You can search in the search bar available on the page, and choose commits in the filter to the left. Im not sure if it will work with commit-id, but it works with ticket name, for example.
<img width="232" height="285" alt="Screenshot 2026-01-23 at 09 55 57" src="https://github.com/user-attachments/assets/7906101a-027e-42dc-b48c-7b9a3a61afdf" />
